### PR TITLE
make search more prominent

### PR DESCRIPTION
**Pull request by eshellman**
## What does this PR do?
 Improve repo

### Why is this valuable (or not)?
existing link does not make clear you can search

### CODE REVIEW

**Comment by LuigiImVector** [comment]
One way to make it even better might be this (perhaps even hunting down the visible link):

image

link to [forked-repo](https://github.com/LuigiImVector/free-programming-books/blob/readme-update-1/README.md)

File: README.md [hunk 1]
```diff
@@ -8,7 +8,9 @@
...
+Search the list at [https://ebookfoundation.github.io/free-programming-books-search/](https://ebookfoundation.github.io/free-programming-books-search/).  [![https://ebookfoundation.github.io/free-programming-books-search/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=Dynamic%20search%20site&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books-search%2F)](https://ebookfoundation.github.io/free-programming-books-search/)
```
Comment (LuigiImVector) [inline README.md:11]:
```suggestion
Search the list at [![https://ebookfoundation.github.io/free-programming-books-search/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=Dynamic%20search%20site&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books-search%2F)](https://ebookfoundation.github.io/free-programming-books-search/).
```

keep the link or not?

Comment (eshellman) [reply README.md:11]: keep it

**Review by LuigiImVector — APPROVED** [review]
Looks good to me

**Review by davorpa — APPROVED** [review]

**Pull request merged**